content/en/spec/security.md

```diff
@@ -11,7 +11,7 @@ menu:
 
 {{< caption-link url="https://github.com/mastodon/mastodon/blob/master/app/lib/request.rb" caption="app/lib/request.rb" >}}
 
-[HTTP Signatures](https://w3c-dvcg.github.io/http-signatures/) is a specification for signing HTTP messages by using a \`Signature:\` header with your HTTP request. Mastodon requires the use of HTTP Signatures in order to validate that any activity received was authored by the actor generating it. When secure mode is enabled, all GET requests require HTTP signatures as well.
+[HTTP Signatures](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-message-signatures-15) is a specification for signing HTTP messages by using a \`Signature:\` header with your HTTP request. Mastodon requires the use of HTTP Signatures in order to validate that any activity received was authored by the actor generating it. When secure mode is enabled, all GET requests require HTTP signatures as well.
```

Comment:
Wrong link, Mastodon is currently using a different draft version: https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures

Reply:
The linked draft is the successor to the deprecated one used by Mastodon as can be seen here: https://datatracker.ietf.org/doc/draft-ietf-httpbis-message-signatures/